src/DataFramesMeta.jl

```diff
@@ -1,6 +1,7 @@
 module DataFramesMeta
 
-using DataFrames
+using DataFrames 
+import Tables: allocatecolumn
```

Comment:
`import` is for when you want to override a function. Here it would be `using`. But since it's used only once better fully qualify it.

Reply:
fixed.